library/src/main/java/com/leinardi/android/speeddial/SpeedDialView.java

```diff
@@ -872,6 +941,7 @@ protected InstanceState(Parcel in) {
             this.mMainFabCloseBackgroundColor = in.readInt();
             this.mExpansionMode = in.readInt();
             this.mMainFabCloseRotateAngle = in.readFloat();
+            this.mUseReverseAnimationOnClose = in.readByte() != 0;
```

Comment:
The value is not restored in `onRestoreInstanceState()`, so the settings is lost after, for example, a rotation.

- [ ] Restore the value in `onRestoreInstanceState()`

Reply:
This is solved, although I didn't check it before my last commit, so it won't stay checked :sweat_smile: